packages/table/src/cell/cell.tsx

```diff
@@ -102,7 +102,15 @@ export class Cell extends React.Component<ICellProps, {}> {
             },
         );
 
-        const content = <div className={textClasses}>{this.props.children}</div>;
+        const modifiedChildren = React.Children.map(this.props.children, (child) => {
+            if (React.isValidElement(child)) {
+                return React.cloneElement(child as React.ReactElement<any>,
```

Comment:
use `JSX.Element` in place of `React.ReactElement<any>` if you don't need props type